assets/i18n/strings/pt_BR.xlb

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<localizationbundle locale="pt-BR">
+<messages>
+
+
+
+
+<msg desc="This title text introduces people to the idea that Google helps them get more (free articles)." hidden="0" name="GAA_REGWALL_TITLE" xml:space="default">Veja mais com o Google</msg>
+<msg desc="This description text tells people they are out of free articles from a given publication. Then it tells people they can unlock more free articles if they create a publisher account with their Google account." hidden="0" name="GAA_REGWALL_DESCRIPTION" xml:space="default"><strong></strong>This article usually requires payment, but you can keep reading, compliments of Google, when you register with <ph name="PUBLICATION"><ex>AP News</ex>{publication}</ph> using your Google account.</msg>
```

Comment:
I see that this msg is in English whereas the other two are in Portuguese. Is that expected? The same thing is true with fr.xlb.

Reply:
Yep! This message is newer and hasn't been translated yet